eventemitter2.d.ts

```diff
@@ -11,12 +11,12 @@ declare module eventemitter2 {
          * @default '.'
          * @description the delimiter used to segment namespaces.
          */
-        delimiter?: string, 
```

Comment:
The change is in the emitAsync method parameters.
Was missing the additional argument "values" in the method.